helper/lib/src/service/algolia_multi_search_service.dart

```diff
@@ -1,12 +1,11 @@
 import 'package:algoliasearch/algoliasearch.dart' as algolia;
 import 'package:logging/logging.dart';
 
-import '../lib_version.dart';
+import '../../algolia_helper_flutter.dart';
```

Comment:
I couldn't figure out the reason behind this new import based on the changes, could you give me more info on this?

Reply:
it is actually better to use `lib_version.dart` instead of importing the whole library using `algolia_helper_flutter, nice catch!